cmd/k8s-operator/operator.go

```diff
@@ -347,14 +374,16 @@ func (a *ServiceReconciler) Reconcile(ctx context.Context, req reconcile.Request
 		return reconcile.Result{}, fmt.Errorf("failed to get device ID: %w", err)
 	}
 	if tsHost == "" {
+		logger.Debugf("no Tailscale hostname known yet, waiting for proxy pod to finish auth")
 		// No hostname yet. Wait for the proxy pod to auth.
 		svc.Status.LoadBalancer.Ingress = nil
 		if err := a.Status().Update(ctx, svc); err != nil {
 			return reconcile.Result{}, fmt.Errorf("failed to update service status: %w", err)
 		}
-		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
Is this cleanup from the previous PR?

Reply:
Cleanup found while working through the code and adding logging. The previous code was mostly harmless, just queued some pointless no-op work for later.

Reply:
👍🏾